about.html

```diff
@@ -41,13 +41,23 @@ <h2>Open Licensing</h2>
   </div>
   <div class="col-sm-6">
     <img src="{{ site.baseurl }}/assets/images/magnify.png"/>
+
     <h2>VuFind® in Action</h2>
-    Take a look at our <a href="https://vufind.org/wiki/community:installations">VuFind® Installations Wiki page</a> to see how a variety of organizations have taken advantage of VuFind®'s flexibility. If you are already using VuFind®, tell us about it through our <a href="https://vufind.org/advanced_demo/Feedback/Form/InstallationReport">Installation Report form</a>.
+
+    <p>Take a look at our <a href="https://vufind.org/wiki/community:installations">VuFind® Installations Wiki page</a> to see how a variety of organizations have taken advantage of VuFind®'s flexibility. If you are already using VuFind®, tell us about it through our <a href="https://vufind.org/advanced_demo/Feedback/Form/InstallationReport">Installation Report form</a>.</p>
+
+    <h2>Active Community</h2>
+
+    <p>VuFind® is maintained by international community of developers. Libraries around the world customize and improve VuFind®, contributing those improvements to the larger community of open-source developers.</p>
```

Comment:
Slight edit suggestion:

VuFind® is maintained by an international community of developers. Libraries around the world customize and improve VuFind®, contributing those improvements back to project to be shared by all.

Reply:
(You're missing "an" in front of "international" -- we definitely want to add that. The adjustments to the second sentence are more subjective, but I thought it might make the language a little less redundant).

Reply:
Much better! I felt redundant from the home page!